README.md

```diff
@@ -96,7 +96,43 @@ not be repeated, and should be placed in a folder named by date.
 Repeated events are always shown at the top of the events page, and 
 do not expire.
 
-### 4. How do I add a page redirect?
+### 4. How do I add a community document?
+
+A community document is a living document that represents a community effort
+to discuss a question or idea. We call it "living" because we encourage community
+members to edit and contribute to the documents over time, and so there isn't
+a "publish" data associated with it. For this reason, we use [the wiki](https://github.com/USRSE/usrse.github.io/wiki) associated
+with the repository here. 
+
+![assets/img/usrse-book-small.png](assets/img/usrse-book-small.png)
+
+If you ever need to clone this content, you can do:
+
+```bash
+$ git clone git@github.com/USRSE/usrse.github.io/wiki
+```
+
+During the testing phase of the living documents, the wiki will only be editable by community (repository)
+members, however we might open this up to include others as well. 
+
+#### How do I edit an existing document?
+
+To edit a living document that exists, you can browse the [wiki](https://github.com/USRSE/usrse.github.io/wiki) to update it. Generally, we are organizing topics as they are added, so if you want to add a new category or change existing organization, please do so. 
+
+#### How do I start a new document?
+
+If you want to add a new document, while you might add it directly to the wiki, it's recommended to get others
+involved first. You can ping others in the USRSE slack to contribute, and it's also recommended to start the document as a Google
+Document for easier commenting and editing, and then move to the wiki when it's in a final first draft state. 
+For all community documents, the following points apply:
+
+1. It's strongly suggested to first work on your document in Google Docs or similar, where you can bring in multiple community members to put together a first draft. Importantly, you should make sure that the content adheres to the US-RSE [Code of Conduct](https://us-rse.org/code-of-conduct/). At least 3 community members (including yourself) are required before submitting the document to the site. You should feel that it's in a solid "first draft state" before doing a submission.
+2. You should verify that 1) the content is of interest to the RSE community, and 2) the content does not violate the Code of Conduct. Any material added to the wiki that does not follow these rules is subject to removal. If you see content on the wiki that is not appropriate or otherwise breaks these rules, please open an issue on the repository immediately, or contact the steering committee.
```

Comment:
I better like this approach of writing 😄 
```
2. Make sure you verify the following:
    - the content is of interest to the RSE community
    - the content does not violate the Code of Conduct

NOTE:  Any material added to the wiki that does not follow these rules is subject to removal. If you see content on the wiki that is not appropriate or otherwise breaks these rules, please open an issue on the repository immediately, or contact the steering committee.
```

Reply:
You got it!